params/bootnodes.go

```diff
@@ -66,6 +66,14 @@ var OdysseyBootnodes = []string{
 	"enode://298bc05889e120e50e7a344e31adb72901caaff9e747b3e8f57c5786e6167452857cbd0ff7d5e0f849a8dfd26efcc3da1795e28d7cb05d120e97a89e79ec4b18@b2.odyssey-testnet.storyrpc.io:30303",
 }
 
+// StoryBootnodes are the enode URLs of the P2P bootstrap nodes running on the
+// Story main network.
+var StoryBootnodes = []string{
+	// Upstream bootnodes
+	"enode://077aa94a5bf388adda35e0d5a7bd8cb5e9741ea5c18381fd14b16018e112497b7c0a400f50677deb4271af3bc0a1b74f093067a398257b76e3a0d7387a432110@b1.odyssey-testnet.storyrpc.io:30303",
+	"enode://298bc05889e120e50e7a344e31adb72901caaff9e747b3e8f57c5786e6167452857cbd0ff7d5e0f849a8dfd26efcc3da1795e28d7cb05d120e97a89e79ec4b18@b2.odyssey-testnet.storyrpc.io:30303",
```

Comment:
I guess they are dummy ones since they are still the same as odyssey bootnodes?

Reply:
Right. As described in this PR, the bootnodes and genesis data are from Odyssey. It will be updated.

Reply:
May want to add a TODO to change it later?

Reply:
Good. done in [49a4eac](https://github.com/piplabs/story-geth/pull/77/commits/49a4eac410f157c70a765bf7aa1b8313774c78ce).